edx_repo_tools/codemods/django3/setup_file_modernizer.py

```diff
@@ -37,7 +39,7 @@ def _add_new_classifiers(self, file_data) -> str:
         modified_file_data = file_data
         for classifier in self.new_trove_classifiers:
             modified_file_data = (modified_file_data[:end_index_of_most_recent_classifier] +
-                                  classifier +
+                                  self.trove_classifiers_indentation + classifier +
```

Comment:
Thinking about it again, it would be better approach to use the `rjust()` function instead.
```suggestion
                                  classifier.rjust(TROVE_CLASSIFIERS_INDENT_COUNT) +
```